scripts/validate_string_concatenation.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+"""
+Check where there is a string that needs to be concatenated.
+"""
```

Comment:
can you flesh this out with a GH reference and a sentence about how `black` causes the issue